ts/store/reducers/index.ts

```diff
@@ -130,7 +127,6 @@ export const appReducer: Reducer<GlobalState, Action> = combineReducers<
   appState: appStateReducer,
   navigation: navigationReducer,
   backoffError: backoffErrorReducer,
-  wallet: walletReducer,
```

Comment:
Be aware that inside this removed instance there was a persisted standalone configuration. We should remove such data from the user device. You can use `ts/sagas/removePersistedStates.ts` by adding the persistence key, which should be `persisted:wallets`

Reply:
Thanks for pointing it out! Addressed into ae81099f158e5d42fa18fb51832982d532c95907